graphene_django/fields.py

```diff
@@ -154,18 +160,61 @@ def resolve_connection(cls, connection, args, iterable, max_limit=None):
         ):
             args["first"] = max_limit
 
+        if graphene_settings.USE_DATALOADERS:
+            if not hasattr(info.context, "dataloaders"):
+                info.context.dataloaders = {}
+
+            dataloader_key: str = str(info.field_nodes)
```

Comment:
Me being ignorant of graphql-core: `field_nodes` is a `list` or a `dict` of objects, right? If so, is stringifying all that efficient, robust, unique enough...?

Maybe irrelevant, but came across this approach [here](https://github.com/graphql-python/graphql-core/blob/0c93b8452eed38d4f800c7e71cf6f3f3758cd1c6/src/graphql/execution/execute.py#L1605C29-L1605C40):

```py
# We cannot use the field_nodes themselves as key for the cache, since they
# are not hashable as a list. We also do not want to use the field_nodes
# themselves (converted to a tuple) as keys, since hashing them is slow.
# Therefore, we use the ids of the field_nodes as keys. Note that we do not
# use the id of the list, since we want to hit the cache for all lists of
# the same nodes, not only for the same list of nodes. Also, the list id may
# even be reused, in which case we would get wrong results from the cache.
key = (
    (return_type, id(field_nodes[0]))
    if len(field_nodes) == 1  # optimize most frequent case
    else tuple((return_type, *map(id, field_nodes)))
)
```

Reply:
Thanks for this @lucas-bremond, super good catch